sormas-backend/src/test/java/de/symeda/sormas/backend/importexport/DatabaseExportServiceTest.java

```diff
@@ -59,11 +65,20 @@ public void testGetConfigFullyDefined() {
 		FirstVaccinationDate.class,
 		AuditLogEntry.class);
 
+	private static final List<String> NOT_EXPORTED_JOIN_TABLE_NAMES = Arrays.asList(
```

Comment:
please create a ticket to don't forget to add these tables too to the database export. Or just do it now :)

Reply:
I've added into the ticket description that these 4 tables should also be added: 
contacts_visits
events_eventgroups
campaign_campaignformmeta
sharerequestinfo_shareinfo

https://github.com/hzi-braunschweig/SORMAS-Project/issues/10185